contracts/cw721-fixed-price/src/contract.rs

```diff
@@ -161,7 +161,7 @@ pub fn execute_receive(
     });
 
     let callback = CosmosMsg::Wasm(WasmMsg::Execute {
-        contract_addr: config.cw20_address.to_string(),
+        contract_addr: config.cw721_address.clone().unwrap().to_string(),
```

Comment:
Can we do this without an `unwrap()` please? :pray:

Reply:
Thanks for the review, it indeed might panic